arangod/Graph/Graph.cpp

```diff
@@ -212,10 +240,31 @@ std::set<std::string> const& Graph::orphanCollections() const {
   return _orphanColls;
 }
 
+std::set<std::string> const& Graph::satelliteCollections() const {
+  return _satelliteColls;
+}
+
 std::set<std::string> const& Graph::edgeCollections() const {
   return _edgeColls;
 }
 
+bool Graph::needsToBeSatellite(std::string edge) const {
```

Comment:
Please move this into GraphEE enterprise code.

The community variant should always return false, no matter what